src/app/app.component.ts

```diff
@@ -5,12 +5,15 @@ import { ConnectionService } from './services/connection.service';
 
 @Component({ selector: 'app-root', templateUrl: 'app.component.html' })
 export class AppComponent implements OnInit {
+
+  public title = $localize`:@@app.title:Sensor Registry`;
```

Comment:
Persoonlijk prefereer ik de cleanere notatie zonder de custom ID
```public title = $localize `Sensor Registry`;```
We hebben het in het verleden daar ook wel over gehad m.b.t. de HTML templates IIRC.
Zonder de custom ID wordt die gegeneerd en kan er ook prima vertaald worden.

Na wat testen kwam ik echter wel een interessante use-case tegen. Dat is wanneer de base text (Engels) aangepast wordt, maar de vertaling(en) gelijk blijven. Zonder custom ID moet je alle vertalingen weer apart toevoegen, die ben je dan kwijt. De custom IDs zijn weliswaar meer code, maar maken de vertalingen wel robuuster. 

Al met al, deze aanpak lijkt me goed. Wellicht geldt hetzelfde dan ook voor de HTML 🤔

Reply:
Lijkt mij prima! Ik denk dat we zodra we wijzigingen aan de code gaan maken er achter gaan komen welke variant voor ons het handigst is ;)